src/C4Config.h

```diff
@@ -267,7 +270,7 @@ class C4Config
 	bool Save();
 	bool Load(bool forceWorkingDirectory = true, const char *szConfigFile = nullptr);
 	bool Init();
-	const char *AtExePath(const char *szFilename);
+	const char *AtExePath(const char *szFilename, bool forceExeFolder=false);
```

Comment:
See other comment above. Also, the style guide mandates spaces next to the operator.